CMakeLists.txt

```diff
@@ -250,7 +253,7 @@ add_subdirectory(cluster)
 add_subdirectory(library)
 add_subdirectory(vowpalwabbit)
 
-if (BUILD_FLATBUFFER_UTILS)
+if (BUILD_FLATBUFFER_UTILS AND BUILD_FLATBUFFERS)
```

Comment:
Should we collapse both of these options into just `BUILD_FLATBUFFERS`? The main reason we have the option is to control whether we require the flatbuffer dependency to be present or not. If it's present I feel like having the target present for fb utils is not an issue

Reply:
Yes good point, will do